uniffi/src/templates/ErrorTemplate.rs

```diff
@@ -0,0 +1,30 @@
+
+#[derive(Debug, Clone, thiserror::Error)]
+enum GenericRustError {
+    #[error("Unknown error")]
```

Comment:
Ditto on the `GenericRustError` Wasn't sure this was the smartest way to do this